module/config.mjs

```diff
@@ -2317,13 +2317,15 @@ DND5E.dieSteps = [4, 6, 8, 10, 12, 20, 100];
  */
 DND5E.damageScalingModes = {
   whole: {
-    label: "DND5E.DAMAGE.Scaling.Whole"
+    label: "DND5E.DAMAGE.Scaling.Whole",
+    labelCantrip: "DND5E.DAMAGE.Scaling.WholeCantrip"
```

Comment:
Just give a little update to the documentation above to indicate this new property exists (`@enumerated {{ label: string, labelCantrip: string }}`).